test/system/500-networking.bats

```diff
@@ -210,6 +210,14 @@ load helpers
 @test "podman network reload" {
     skip_if_remote "podman network reload does not have remote support"
 
+    # reload on container with no ports should not display errors
+    run_podman run -d --network bridge $IMAGE top
+    cid=$output
+
+    run_podman network reload $cid
+    is "$output" "$cid" "output matches container ID"
```

Comment:
Emits no error. However, neither does podman from main. IOW, this test does not actually test anything. What is it intended to test?

Reply:
Oh. Thanks for checking, this must be network connect/disconnect...
I somehow confused this when writing the test.